src/main/java/seedu/duke/financial/FinancialEntry.java

```diff
@@ -23,6 +23,9 @@ public FinancialEntry(double amount, String description, LocalDate date) throws
         if (amount < 0.01) {
             throw new FinanceBuddyException("Invalid amount. Amount must be $0.01 or greater.");
         }
+        if (amount > 9999999.00) {
+            throw new FinanceBuddyException("Invalid amount. Amount must be <= $9999999.00.");
```

Comment:
Can consider standardising the warning message? So both use words or both use < >
But not that important